commitlint.config.ts

```diff
@@ -416,6 +424,26 @@ module.exports = {
                     ];
                 },
 
+                'proper-issue-refs': ({raw}: {raw:any}) => {
+                    // workaround for https://github.com/conventional-changelog/commitlint/issues/3412
```

Comment:
this comment is misplaced; right now it looks like you're referring to the variable "offence", which is very confusing

Reply:
How is it now?

Reply:
better 👍